src/apps/loyalty/server/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as express from "express"
+import * as path from "path"
+import * as React from "react"
+import { renderToString } from "react-dom/server"
+import * as styleSheet from "styled-components/lib/models/StyleSheet"
+import renderPage from "./template"
+
+import Inquiries from "../containers/inquiries"
+import Login from "../containers/login"
+
+const app = express()
+
+app.use(express.static(path.resolve(__dirname)))
+
+app.get("/", (req, res) => {
+  // check if the user is logged in, if not redirect to /login
+  // else render /inquiries
+  res.redirect(req.baseUrl + "/inquiries")
+})
+
+app.get("/login", (req, res) => {
+  const html = renderToString(<Login />)
+  const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
+  res.send(renderPage({
+    styles,
+    html,
+    entrypoint: "/bundles/login.js",
+  }))
+})
+
+app.get("/inquiries", (req, res) => {
+  const user = (req as any).user
```

Comment:
I think integrating as a sub-app, like is being done now, is the right approach, but def agree that the dev workflow will need work after this. I’m +1 on adding artsy-passport (kinda like how we have a simple authentication view in Emission as well) and definitely on adding a TypeScript interface for the API we use that comes in from Force.